Library/Homebrew/cmd/update-report.rb

```diff
@@ -249,6 +249,20 @@ def updated?
   def migrate_tap_migration
     report[:D].each do |full_name|
       name = full_name.split("/").last
+      if tap == "caskroom/cask"
+        next unless (dir = HOMEBREW_REPOSITORY/"Caskroom"/name).exist?
+        next unless new_tap_name = tap.tap_migrations[name] # skip if formula is not in tap_migrations list.
+        new_tap = Tap.fetch(new_tap_name)
+        new_tap.install unless new_tap.installed?
+        ohai "#{name} has been moved to Homebrew Core. Installing #{name}..."
+        system HOMEBREW_BREW_FILE, "install", name
+        ohai <<-EOS.undent
+            To uninstall the cask run:
+              brew cask uninstall --force #{name}
+        EOS
```

Comment:
- The indentation inside the `<<-EOS.undent`–`EOS` block is to deep. It needs to be only be indented by two additional spaces (currently four).

- As mentioned in a previous PR, the first argument of `ohai` is more of a title and subject to truncation and thus isn't really suitable for multi-line output like this. We need either to switch to `puts` or provide a relatively short header like in the migration below from Homebrew to Cask.

  Or maybe with some rearrangement this can be combined with the `ohai` above the `system HOMEBREW_BREW_FILE, "install", name` line.